.stylelintrc.js

```diff
@@ -1,13 +1,19 @@
 module.exports = {
-  extends: ['stylelint-config-wolox'],
+  extends: ["stylelint-config-wolox"],
```

Comment:
- Use single quotes in js,ts or tsx files
```suggestion
  extends: ['stylelint-config-wolox'],
```